api/src/main/resources/workbench-api.yaml

```diff
@@ -558,38 +558,6 @@ paths:
           description: Successfully created auth domain
           schema:
             "$ref": "#/definitions/AuthDomainCreatedResponse"
-  "/v1/auth-domain/users":
-    post:
-      tags:
-      - authDomain
-      consumes:
-      - application/json
-      responses:
-        204:
-          description: Successfully Updated User In Group
-        403:
-          description: You must be an admin of this group in order to enable/disable
-            members
-          schema:
-            "$ref": "#/definitions/ErrorResponse"
-        404:
-          description: User not found
-          schema:
-            "$ref": "#/definitions/ErrorResponse"
-        500:
-          description: FireCloud Internal Error
-          schema:
-            "$ref": "#/definitions/ErrorResponse"
-      parameters:
-      - in: body
-        name: request
-        description: Request containing user email to update and a disabled status
-          to update the user to.
-        schema:
-          "$ref": "#/definitions/UpdateUserDisabledRequest"
-      summary: '[DEPRECATED - use admin/users/updateAccount] enable/disable a user to an auth domain 
```

Comment:
no longer used - the UI transitioned to `admin/users/updateAccount` as part of the admin pages work.